CryptoLib/CryptoLib/XmlParser/XmlParserDelegate.m

```diff
@@ -33,7 +47,7 @@ - (void)parser:(NSXMLParser *)parser didStartElement:(NSString *)elementName nam
 }
 
 -(void) parser:(NSXMLParser *)parser foundCharacters:(NSString *)string {
-    if(string != nil && [string length] != 0 && ![string isEqualToString:@"\n    "] && ![string isEqualToString:@"\n"]){
+    if (string != nil && [string length] != 0 && ![string isEqualToString:@"\n    "] && ![string isEqualToString:@"\n"]){
```

Comment:
This line needs an explanation with the comment.
If "@\n     " and "@\n" are special-cases that needs to be handled here, code reader want to understand why.